.gitignore

```diff
@@ -65,6 +65,7 @@ apis/r/src/libtiledbsoma/*
 apis/r/src/Makevars
 apis/r/src/cmake_log.txt
 apis/r/.editorconfig
+!apis/r/man/roxygen/*
```

Comment:
What does this do? I have never seen a leading `!` in `.gitignore`, nor have I seen man/roxygen/* files.